bundles/org.smarthomej.transform.basicprofiles/README.md

```diff
@@ -0,0 +1,62 @@
+# Basic Profiles
+
+This bundle provides a list of useful Profiles.
+
+## Battery Low Profile
```

Comment:
Yes, of course. I think this will end I the same discussion I had with Kai about this. The result was the Hysteresis Profile enhanced with an `inverted` configuration parameter. Long story short: In general this Profile is just a shortcut for another existing Profile.

Reply:
I really like it, because it‘s so simple and easy to understand. Together with the invert profile and the chain profile (see my PR) you can use it in any way. No need for more configuration. It‘s just the name.

Reply:
:+1: Makes sense. I will change it.

Reply:
> Together with the invert profile and the chain profile (see my PR) you can use it in any way.

This is not clear to me. You Chain Profile only supports chaining Transformations. Did I miss something?

Reply:
You are right. I was think of chaining profiles but that is a different issue from chaining transformations. Interesting idea though.